docs/modules/ROOT/pages/how-to-guides/configuring-builds/proc_customize_build_pipeline.adoc

```diff
@@ -41,27 +41,58 @@ Example task:
 +
 [source]
 --
-name: example-task
-      params:
-      - name: example-param
-        value: “Example”
-      runAfter:
-      - build-container #You can be more specific by choosing another task
-      taskRef:
-        params:
-        - name: name
-          value: example-task
-        - name: bundle
-          value: <path to container image>
-        - name: kind
-          value: task
-        resolver: bundles
-      when:
-      - input: $(params.skip-checks)    #This references the pipeline parameters
-        operator: in
-        values:
-        - "false"
+  name: example-task
+  params:
+  - name: example-param
+    value: “Example”
+  runAfter:
+  - build-container #You can be more specific by choosing another task
+  taskRef:
+    params:
+    - name: name
+      value: example-task # metadata.name field of the Task
+    - name: bundle
+      value: quay.io/tekton-bundle-catalog/example-task-bundle:1.0
+      # For more details on tekton bundles, refer https://tekton.dev/docs/pipelines/pipelines/#tekton-bundles
+    - name: kind
+      value: task
+    resolver: bundles
+  when:
+  - input: $(params.skip-checks)    #This references the pipeline parameters
+    operator: in
+    values:
+    - "false"
 --
+
++
+An example of custom task added to the pipeline which sends slack notification when the pipelinerun fails:
++
+[source]
+--
+  name: slack-webhook-notification
+  params:
+    - name: message
+      value: PipelineRun $(context.pipelineRun.name) failed
+    - name: secret-name
+      value: my-secret # name of the secret in the your namespace which contains slack web-hook URL
```

Comment:
I would add something like `... slack web-hook URL under key specified in 'key-name' parameter below`

Reply:
Added now.